install/kubernetes/cilium/files/nodeinit/startup.bash

```diff
@@ -23,6 +23,11 @@ if ip link show cbr0; then
 fi
 {{- end }}
 
+{{- if .Values.nodeinit.removeAWSCNI }}
+echo "Removing stale AWS iptables rules..."
+sudo iptables -L
```

Comment:
iirc this code executes within the context of pid1 on the host, so the availability of `sudo` here is not a given, and there's no point to using it. Also, how does `iptables -L` remove iptables rules?